docs/administrators/logging.rst

```diff
@@ -86,59 +63,98 @@ file can be found at ``/usr/lib/systemd/system/inmanta-server.service``.
   Type=simple
   User=inmanta
   Group=inmanta
-  ExecStart=/usr/bin/inmanta --log-file /var/log/inmanta/server.log --log-file-level 2 --timed-logs server
+  ExecStart=/usr/bin/inmanta --log-file /var/log/inmanta/server.log --log-file-level INFO --timed-logs server
   Restart=on-failure
 
   [Install]
   WantedBy=multi-user.target
 
-The ``--log-file-level`` takes the log-level as an integer, where ``0=ERROR``, ``1=WARNING``, ``2=INFO`` and ``3=DEBUG``.
-
-Apply the changes by reloading the service file and restarting the Inmanta server:
 
 .. code-block:: sh
 
   sudo systemctl daemon-reload
   sudo systemctl restart inmanta-server
 
 
-Log level manually started agent
---------------------------------
+Fine grained configuration
+----------------------------
+For fine grained configuration, `a standard python dict config file <https://docs.python.org/3/library/logging.config.html#logging-config-dictschema>`_ can be passed in via the config file for each component individually:
+
+.. code-block:: toml
+
+    [logging]
+    server = server_log.yml
+    scheduler = scheduler.log.tmpl
+    compiler = compiler.yml
```

Comment:
This gives a warning at build time, because it's not valid toml. I am not sure, but you probably have to quote the string values to make if valid toml.